### PR TITLE
Feature/#406 conversion API client reimplementation: convert existing tests to py.test

### PR DESCRIPTION
**Pull request by das-g**
Some of this will have to change again for #406, but having this change here already reviewed & merged will keep future PR's tidier and easier to review.

### Reviewed by
- [x] @hixi

(Loosely) connected to #406

### CODE REVIEW

File: tests/excerptexport/test_conversion_api_client.py [hunk 1]
```diff
@@ -17,222 +16,249 @@
...
-    # TODO: re-record VCR (service API has changed)
 
-    @vcr.use_cassette('fixtures/vcr/conversion_api-test_create_job.yml')
-    def test_create_job(self):
-        self.assertIsNone(self.extraction_order.process_id)
+#
+# ConversionApiClient integration tests:
 
-        response = self.api_client.create_job(self.extraction_order, request=self.request)
+# TODO: re-record VCR (service API has changed)
```
Comment (hixi) [inline tests/excerptexport/test_conversion_api_client.py]: :+1:

File: tests/excerptexport/test_conversion_api_client.py [hunk 2]
```diff
@@ -17,222 +16,249 @@
...
-    def test_order_status_processing(self):
-        cassette_file_location = os.path.join(
-            absolute_cassette_lib_path,
-            'fixtures/vcr/conversion_api-test_order_status_processing.yml'
+        content_types_of_output_files = (f.content_type for f in extraction_order.output_files.all())
+        ordered_formats = extraction_order.extraction_configuration['gis_formats']
+        assert_that(content_types_of_output_files, contains_inanyorder(*ordered_formats))
+        assert_that(
+            len(extraction_order.output_files.order_by('id')[0].file.read()),
+            is_close_to(446005, delta=10000)
```
Comment (hixi) [inline tests/excerptexport/test_conversion_api_client.py]: much better to understand :-)

**Comment by hixi** [comment]
very nice :+1:

File: tests/excerptexport/test_conversion_api_client.py (hunk 1 above)
Comment (das-g) [inline tests/excerptexport/test_conversion_api_client.py]: Actually, this comment was here [before this PR](https://github.com/geometalab/osmaxx-frontend/pull/426/files/d61c77df4cf6d7ac58132747b2dac9fc0c957964#diff-b7bf2d14794ca379596eebbdc057b222L109) already.

**Pull request merged**